Gopkg.toml

```diff
@@ -40,6 +40,12 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.13.1"
 
+[[constraint]]
+  name = "k8s.io/api/coordination/v1beta1"
+  source="https://github.com/kubernetes/api.git"
+  branch="master"
+  version = "kubernetes-1.13.0"
```

Comment:
Wouldn't we make this 1.13.1 to match everything else from k8s?

Reply:
done